fbw-common/src/wasm/systems/systems/src/hydraulic/flap_slat.rs

```diff
@@ -179,19 +263,16 @@ impl FlapSlatAssembly {
         circuit_target_pressure: Pressure,
     ) -> Self {
         Self {
-            position_left_percent_id: context
-                .get_identifier(format!("LEFT_{}_POSITION_PERCENT", id)),
-            position_right_percent_id: context
-                .get_identifier(format!("RIGHT_{}_POSITION_PERCENT", id)),
-
-            angle_left_id: context.get_identifier(format!("LEFT_{}_ANGLE", id)),
-            angle_right_id: context.get_identifier(format!("RIGHT_{}_ANGLE", id)),
-
             animation_left_id: context.get_identifier(format!("LEFT_{}_ANIMATION_POSITION", id)),
             animation_right_id: context.get_identifier(format!("RIGHT_{}_ANIMATION_POSITION", id)),
 
+            ippu_id: context.get_identifier(format!("{}_IPPU_ANGLE", id)),
+            fppu_id: context.get_identifier(format!("{}_FPPU_ANGLE", id)),
```

Comment:
```suggestion
            ippu_id: context.get_identifier(format!("{id}_IPPU_ANGLE")),
            fppu_id: context.get_identifier(format!("{id}_FPPU_ANGLE")),
```
Yes I know the current code is not doing it this way but I think we still should do it.